mne/io/base.py

```diff
@@ -363,6 +368,26 @@ def __init__(self, info, preload=False,
         self._comp = None
         self._filenames = list(filenames)
         self.orig_format = orig_format
+        # Sanity check and set original units, if provided by the reader:
+        if orig_units:
```

Comment:
`if orig_units is not None` ? Otherwise, this check will go through if I give `orig_units=True`

Reply:
thanks! I will raise an error if orig_units is not None and not a dict.